<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Order_information</class>
 <widget class="QWidget" name="Order_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>710</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="order">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1221</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>查询方式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchSelect">
      <item>
       <property name="text">
        <string>默认</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>订单号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>游客姓名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>时间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>套餐</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>导游姓名</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirm">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>184</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>总收入为：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="income"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>304</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>设置景区游客数量：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_2">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
